src/DateRange/DateRange.tsx

```diff
@@ -0,0 +1,291 @@
+							hours={timeFrom.hours}
+							minutes={timeFrom.minutes}
+							noShadow={false}
+							onChange={handleChangeTimeFrom}
+						/>
+					</div>
+				</div>
+			</div>
+			<div className="kit-date-range__popup-dates">
+				<span className="kit-date-range__popup-date-text">
+					{radioConcreteToText}
+				</span>
+				<div className="kit-date-range__popup-fields">
+					<div className="kit-date-range__popup-date">
+						<DateField
+							error={validationDates() === PropError.Date}
+							defaultDate={dateTo}
+							noShadow={false}
+							onChange={handleChangeDateTo}
+						/>
+					</div>
+					<div className="kit-date-range__popup-date">
+						<TimeField
+							error={validationDates() === PropError.Time}
+							hours={timeTo.hours}
+							minutes={timeTo.minutes}
+							noShadow={false}
+							onChange={handleChangeTimeTo}
+						/>
+					</div>
+				</div>
+			</div>
+			{!!validationDates() && (
+				<div className="kit-date-range__popup-tooltip">
+					<Tooltip title={<IconSvg type="warning" />} position="top">
+						Левая граница интервала должна быть меньше правой
+					</Tooltip>
+				</div>
+			)}
+		</div>
+	);
+
+	const handleCancelFilterButtonClick = () => {
+		setDateFrom(defaultDateFrom);
+		setDateTo(defaultDateTo);
+		setTimeFrom(defaultTimeFrom);
+		setTimeTo(defaultTimeTo);
+
+		onCloseFilter();
+	};
+
+	return (
+		<div className="kit-date-range">
+			<div className="kit-date-range__content">
+				<label className="kit-date-range__label">{labelNoFilter}</label>
+				<RadioButton
+					name="date"
+					onSelected={handleSelected(DateRangeValueTypes.NoFilter)}
+					checked={value === DateRangeValueTypes.NoFilter}
+				>
+					{radioTextNoFilter}
+				</RadioButton>
+			</div>
+			<div className="kit-date-range__content">
+				<label className="kit-date-range__label">{labelConcrete}</label>
+				<div className="kit-date-range__radio-button">
+					<RadioButton
+						name="date"
+						onClick={handleToggleFilter(
+							DateRangeValueTypes.Concrete
+						)}
+						checked={value === DateRangeValueTypes.Concrete}
+					>
+						{`${radioConcreteFromText} ${parseDateToString(
+							dateFrom
+						)}
+					${radioConcreteToText} ${parseDateToString(dateTo)}`}
+					</RadioButton>
+
+					{isShowFilter && (
+						<WithOutsideClickFilterDetails
+							editorComponent={
+								<FilterConditionEditorComponent
+									innerEditorComponent={
+										<InnerEditorComponent />
+									}
+									viewMode="edit"
+									addFilterButtonCaption="Применить изменения"
+									cancelFilterButtonCaption="Отменить"
+									isAddFilterButtonEnabled={true}
+									disabledSaveButton={!!validationDates()}
+									onAddFilterButtonClick={handleApplyFilter}
+									onCancelFilterButtonClick={
+										handleCancelFilterButtonClick
+									}
+								/>
+							}
+							viewMode="edit"
+							onClickOutside={onCloseFilter}
+							helpCaption={helpCaption}
+							onClose={onCloseFilter}
```

Comment:
И тут, соответственно
```suggestion
							onClose={handleCancelFilterButtonClick}
```